src/utils/getDataGroupBy.ts

```diff
@@ -1,3 +1,4 @@
+import { Obj } from '@mongez/reinforcements';
```

Comment:
You may not have to involve this package as se already have `lodash` as a dependency and you can use its `get` method.

Reply:
@SevenOutman 

I removed the package and updated the package-lock.json file

Have a check on it now.

Regards,